src/components/RegistrationForm/Form/fields/ValidatePasswordBox/ValidatePasswordBox.jsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import { StyledIcon, StyledItem, StyledValidatePasswordBox } from './ValidatePasswordBox.styled.js';
+
+const ValidatePasswordBox = ({ validationPasswordResults }) => {
```

Comment:
That's a little strange name. That can be for instance ValidationErrors or ErrorsList etc.

Reply:
rename to ValidateErrors